### PR TITLE
Ward Button

### PR DESCRIPTION
**Pull request by Chronophylos**
Disabled Scan Button for now.
Drops a ward at your feet.

### CODE REVIEW

File: game/scripts/vscripts/components/glyph/glyph.lua [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  local target = EntIndexToHScript(keys.entindex_target)
+
+  if order == DOTA_UNIT_ORDER_GLYPH then
+    -- Handle Glyph aka Ward Button
+    DebugPrintTable(keys)
+    self:CastWard(issuerID)
+    return false
+  elseif order == DOTA_UNIT_ORDER_RADAR then
+    -- Handle Scan
+    DebugPrintTable(keys)
```
Comment (chrisinajar) [inline game/scripts/vscripts/components/glyph/glyph.lua]: Does this key contain the location they chose?

Comment (Chronophylos) [inline game/scripts/vscripts/components/glyph/glyph.lua]: Yes

Comment (Chronophylos) [inline game/scripts/vscripts/components/glyph/glyph.lua]: I think it was 
`position_x`, `position_y`, `position_z`

**Comment by Chronophylos** [comment]
I forgot to display the ward cooldown...

**Pull request merged**